doc/interfaces/supercomputer_data/regions.md

```diff
@@ -0,0 +1,155 @@
+---
+title: "Regions"
+filters:
+  - "include-code-files"
+---
+
+
+
+## Regions
+
+Any region that can be selected for viewing on the map or plot.
+Includes Super Regions and Sub Regions.
+
+- All regions (Super Regions and Sub Regions) **MUST** have a unique numeric region
+  identifier.
+    - Regions and Region Collections **MUST NOT** be identified by any of the following
+      reserved strings:
+        - `undefined`
+        - `regions`
+        - `collections`
+        - `metadata`
+
+
+### Super Regions
+
+These are top-level regions, e.g. "Western U.S.", "Canada".
+The webapp needs to be able to quickly load the list of super-regions so it can display
+a selector to the user. Depending on which is selected, different sub-regions may
+display.
+
+- Super Regions definition file **MUST** be named `root.json`.
+- Super Regions definition file **MUST** be in compliance with the schema.
+- Super Regions definition file **MUST** be pushed to the `regions/` subdirectory of the
+  incoming directory, e.g. `{incomingDir}/regions/root.json`.
+- Super Regions **MAY** be omitted from the Super Regions definition file if no variable
+  are visible for the super region.
+
+  :::{.callout-note}
+  In a future version of the schema, we should offer a boolean config option `enabled`
+  and recommend setting it to false to omit a region.
+  :::
+
+- Super Regions with no Sub Regions/Collections **MAY** be included in the Super Regions
+  definition file.
+
```

Comment:
- Super regions were defined as a group of Modis tiles handled in a similar way on the supercomputer side, that is with the same version of the filtering and interpolating algorithm, and the same configuration parameters.
- Some super regions can have identical geotiff input files, for instance western Canada and Alaska.

Reply:
>Super regions were defined as a group of Modis tiles handled in a similar way on the supercomputer side, that is with the same version of the filtering and interpolating algorithm, and the same configuration parameters.

How does that need to be reflected in this interface specification? For the purposes of the interface, it seems to me like this might not be significant?

>Some super regions can have identical geotiff input files, for instance western Canada and Alaska.

Does 21cac1e address this comment?

Reply:
I think it's best to be kept because this has an impact on everything, even if it should be transparent for the webapp.

Second, yes 21cac1e addresses the comment